src/main/java/hudson/plugins/promoted_builds/conditions/ManualCondition.java

```diff
@@ -2,23 +2,14 @@
 
 import hudson.EnvVars;
 import hudson.Extension;
-import hudson.model.AbstractBuild;
-import hudson.model.AbstractProject;
-import hudson.model.Descriptor;
-import hudson.model.Hudson;
-import hudson.model.InvisibleAction;
-import hudson.model.ParameterDefinition;
-import hudson.model.ParameterValue;
-import hudson.model.Run;
-import hudson.model.TaskListener;
-import hudson.model.User;
+import hudson.model.*;
```

Comment:
Please do not squash imports in the production code.
Your IDE can be reconfigured to disable it